src/components/Compare/Filters.tsx

```diff
@@ -101,14 +108,23 @@ const Filters = (props: {
 
   const switchHandleChange = () => {
     if (props.isHomepage && props.setViewAllCounties) {
-      props.setViewAllCounties(!props.viewAllCounties);
+      const newViewAllCounties = !props.viewAllCounties;
+      props.setViewAllCounties(newViewAllCounties);
+      trackCompareEvent(
+        EventAction.SELECT,
+        `Location Type: ${newViewAllCounties ? 'Counties' : 'States'}`,
+      );
     }
   };
 
   const sliderHandleChange = (event: any, value: any) => {
     if (props.setGeoScope) {
       setSliderValue(value);
       props.setGeoScope(sliderNumberToFilterMap[value]);
+      trackCompareEvent(
+        EventAction.SELECT,
+        `GeoScope: ${trackingGeoScopeLabel[sliderNumberToFilterMap[value]]}`,
```

Comment:
FWIW- I think you could do `GeoScopeFilter[sliderNumberToFilterMap[value]]` and it would return 'NEARBY' or whatever and you wouldn't need tracking GeoScopeLabel if we were okay just using the enum names.

Basically, TypeScript enums are a bidirectional map of the enum value<=>enum name

But this is also fine as-is if you just want to ship it.